## Урок 1. Случайные события. Условная вероятность. Формула Байеса. Независимые испытания

#### 1. Из колоды в 52 карты извлекаются случайным образом 4 карты:
<ol> <li>  Найти вероятность того, что все карты – крести. </li>
    <li> Найти вероятность, что среди 4-х карт окажется хотя бы один туз.</li></ol>

In [17]:
four_of_clubs = (13/52) * (12/51) * (11/50) * (10/49) * 100
at_least_one_ace = (1 - (48/52) * (47/51) * (46/50) * (45/49)) * 100 ## 1 минус вероятность 0 тузов из 4х карт
four_of_clubs, at_least_one_ace

(0.26410564225690275, 28.12632745405854)

##### Ответ: <br>все карты крести ~ 0.26 % <br>  хотя бы один туз ~ 28.13 %

#### 2. На входной двери подъезда установлен кодовый замок, содержащий десять кнопок с цифрами от 0 до 9. <br> Код содержит три цифры, которые нужно нажать одновременно. <br> Какова вероятность того, что человек, не знающий код, откроет дверь с первой попытки?

In [10]:
all_combinations = (10*9*8)/(3*2)
one_prob = 1 / all_combinations * 100
one_prob

0.8333333333333334

##### Ответ: <br> вероятность открыть дверь с первой попытки ~ 0.83 %

#### 3. В ящике имеется 15 деталей, из которых 9 окрашены. Рабочий случайным образом извлекает 3 детали.  <br> Какова вероятность того, что все извлеченные детали окрашены?

In [12]:
(9/15) * (8/14) * (7/13) * 100

18.461538461538456

##### Ответ: <br> 3 извлеченные детали окрашены ~ 18.46 %

#### 4. В лотерее 100 билетов. Из них 2 выигрышных. <br>Какова вероятность того, что 2 приобретенных билета окажутся выигрышными?

In [15]:
(2/100) * (1/99) * 100

0.020202020202020204

##### Ответ: <br>  оба билета выигрышные ~ 0.02 %